storybook/src/elements/nuxeo-video/nuxeo-video-conversions.stories.js

```diff
@@ -0,0 +1,20 @@
+import { html } from 'lit-html';
+import { storiesOf } from '@storybook/polymer';
+import { text } from '@storybook/addon-knobs';
+import '@nuxeo/nuxeo-ui-elements/nuxeo-video/nuxeo-video-conversions';
+import '@nuxeo/nuxeo-ui-elements/widgets/nuxeo-card';
+import DocumentBuilder from '../../data/documents.data';
+import videoProperties from '../../data/video.data.js';
+
+storiesOf('UI/nuxeo-video', module).add('Nuxeo Video Converter', () => {
+  const documentBuilder = new DocumentBuilder()
+    .setType('File')
+    .setProperties({ 'vid:transcodedVideos': videoProperties['vid:transcodedVideos'] });
+  documentBuilder.build();
+  const label = text('Label', 'Video Conversion');
+  return html`
+    <nuxeo-card>
+      <nuxeo-video-conversions .document="${documentBuilder}" label="${label}"> </nuxeo-video-conversions>
```

Comment:
Using here the documentBuilder instance works by luck 🙂 i.e., because the way builder stores properties is similar to what a document does.

I would suggest the following:
`const document = new DocumentBuilder()
    .setType('File')
    .setProperties({ 'vid:transcodedVideos': videoProperties['vid:transcodedVideos'] })
    .build();`

And then we'll use `.document="${document}"`

Other possibility, would be to remove the line 13 (`documentBuilder.build()`) since we are not doing nothing with it's output, and map the property as: `.document="${documentBuilder.build()}"`

Reply:
changed the instance name to document as suggested